Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine
```

Comment:
I wanted to use `scratch` but was receiving the following error during `CMD [/bin/sh]`:

```
exec: "/bin/sh": stat /bin/sh: no such file or directory
```